tests/console/docker.pm

```diff
@@ -28,14 +28,29 @@ use testapi;
 use utils;
 use strict;
 use registration;
+use version_utils qw(is_sle is_leap);
+
+sub test_seccomp {
+    my $no_seccomp = script_run('docker info | tee /dev/tty | grep seccomp');
+    if ($no_seccomp) {
+        my $err_seccomp_support = 'boo#1072367 - Docker Engine does NOT have seccomp support';
+        if (is_sle('<15') || is_leap('<15.0')) {
+            record_info('WONTFIX', $err_seccomp_support);
```

Comment:
Just an opinion, but I would not keep WONTFIX in tests - great to have review from X people, right? :)
I think we can merge.